### PR TITLE
FP21-70-footer-new-design

### PR DESCRIPTION
**Pull request by veronicaburduzhan**
# Description

This PR contains the latest version of the design for the footer. 

Fixes # (update css)

# How to test?

Run client and check out localhost:3000. See the pictures of new footer designs below. 
Screenshot 2022-07-22 at 20 53 30
Screenshot 2022-07-22 at 20 54 18
Screenshot 2022-07-22 at 20 55 09
Screenshot 2022-07-22 at 20 54 31
Screenshot 2022-07-22 at 20 54 50
Screenshot 2022-07-22 at 20 54 58

# Checklist

- [x] I have performed a self-review of my own code
- [x] I have followed the name conventions for CSS Classnames and filenames, Components names and filenames, Style filenames, if you are in doubt check the the project README.MD and here https://github.com/HackYourFuture-CPH/curriculum/blob/master/review/review-checklist.md
- [x] I have commented my code, particularly in hard-to-understand areas, if you code was simple enough mark the box anyway
- [x] I have made corresponding changes to the documentation, if you code was simple enough mark the box anyway
- [x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

**Comment by miltonhultgren** [comment]
Could you include a picture in the PR description of what the old design looks like and what the PR code makes it look like?
It's generally a good idea to include such information in a PR, it means I don't have to run the code just to see what it looks like.

File: packages/client/src/components/Footer/Footer.component.js [hunk 1]
```diff
@@ -4,43 +4,57 @@ import { Link } from 'react-router-dom';
...
+    <footer>
+      <div className="footer-bg">
+        <div className="footer-container">
+          <div className="copyright-disclaimer-container">
+            <p className="copyright-disclaimer-text">
+              Copyright Kaospilot. All methods & tools © their respective
+              owners.
+            </p>
+            <p className="copyright-disclaimer-text">
+              Are you an owner to a method/tool, please contact us.
```
Comment (miltonhultgren) [inline packages/client/src/components/Footer/Footer.component.js]: Is it worth to add a link here to maybe a mailto?

File: packages/client/src/components/Footer/Footer.component.js [hunk 2]
```diff
@@ -4,43 +4,57 @@ import { Link } from 'react-router-dom';
...
+            </p>
+          </div>
+          <div className="links-container">
+            <div>
+              <Link to="/">
+                <p className="logo-link-text">KAOSPILOT</p>
+              </Link>
+            </div>
+            <div className="social-media-logo-container">
+              <Link to="/">
```
Comment (miltonhultgren) [inline packages/client/src/components/Footer/Footer.component.js:42]: Do we not have the real URLs for these?

File: packages/client/src/components/Footer/Footer.style.css [hunk 3]
```diff
@@ -1,176 +1,223 @@
...
+
+.footer-bg {
+  background-color: rgba(0, 0, 0, 0.3);
+  mix-blend-mode: normal;
 }
 
 .footer-container {
-  background-color: rgba(40, 48, 49);
-  flex-direction: column;
+  width: 1220px;
```
Comment (miltonhultgren) [inline packages/client/src/components/Footer/Footer.style.css:52]: How does this react in smaller sizes of the screen?
I also see a 613px below, why aren't these using percentages to adjust and then let the breakpoints change which percentage to use? 

That's at least what I am used to doing :)

File: packages/client/src/components/Footer/Footer.style.css [hunk 4]
```diff
@@ -1,176 +1,223 @@
...
+.copyright-disclaimer-text,
+.find-information-text {
+  font-weight: 400;
+  font-size: 1.125rem;
+  padding: 0.625px 0.438rem;
 }
 
-.placeholder-text {
+.links-container {
+  gap: 1.75rem;
```
Comment (miltonhultgren) [inline packages/client/src/components/Footer/Footer.style.css:86]: Generally, I wouldn't mix rem and px. Pick one and stick to it for things like spacings. 

Are you aware of how they differ?

**Review by miltonhultgren — APPROVED** [review]
I think this looks mostly fine!

File: packages/client/src/components/Footer/Footer.style.css (hunk 4 above)
Comment (veronicaburduzhan) [reply packages/client/src/components/Footer/Footer.style.css:86]: > Could you include a picture in the PR description of what the old design looks like and what the PR code makes it look like? It's generally a good idea to include such information in a PR, it means I don't have to run the code just to see what it looks like.

I added screenshots of the new design. The previous design wasn't related to our project. We just recreated the one from the Kaospilot website to have at least something :)

**Comment by veronicaburduzhan** [comment]
> Could you include a picture in the PR description of what the old design looks like and what the PR code makes it look like? It's generally a good idea to include such information in a PR, it means I don't have to run the code just to see what it looks like.

I added screenshots of the new design. The previous design wasn't related to our project. We just recreated the one from the Kaospilot website to have at least something :)

File: packages/client/src/components/Footer/Footer.style.css (hunk 4 above)
Comment (veronicaburduzhan) [reply packages/client/src/components/Footer/Footer.style.css:86]: Yes, I know. I just didn't know that it was a bad practice to use both of the for spacings. Thanks :)

File: packages/client/src/components/Footer/Footer.component.js (hunk 1 above)
Comment (alina-kho) [reply packages/client/src/components/Footer/Footer.component.js]: Agree with Milton on that. Maybe a mailto linked to the 'contact us' could be a good idea

**Review by alina-kho — APPROVED** [review]
Looks really good 👍🏼 tried changing device dimensions - works flawlessly ☺️ 

Will need to update SM links in the future though.

**Review by shpomp — APPROVED** [review]

File: packages/client/src/components/Footer/Footer.style.css (hunk 3 above)
Comment (veronicaburduzhan) [reply packages/client/src/components/Footer/Footer.style.css:52]: Sorry for answering this question for so long. 
1) smaller sizes start from 1300px, when a user gets this point he has 90vw width all the time. All screens wider than 1300px should have 1220px of the footer as this is the max size 
2) below this footer container I made some sizes from px to % that change their width depending on screen size and left one that always stays the same

File: packages/client/src/components/Footer/Footer.component.js (hunk 1 above)
Comment (veronicaburduzhan) [reply packages/client/src/components/Footer/Footer.component.js]: Done✨

**Pull request merged**